src/Uno.Playground.Shared/Styles/Controls/CommandBar.xaml

```diff
@@ -177,8 +177,8 @@
 		</Setter>
 	</android:Style>
 
-	<!--Default CommandBar Style-->
+	<!--Default CommandBar Style--><!--
```

Comment:
Can you explain why it's commented here please ;) ? 
Don't need it anymore ? Related to an issue ?

Reply:
Similar here 👍

Reply:
Let's remove it then.